pkg/services/store/object/object.proto

```diff
@@ -0,0 +1,242 @@
+syntax = "proto3";
+package object;
+
+option go_package = "./;object";
+
+// Will be replaced with something from the SDK
+message UserInfo {
+  int64 id = 1; // internal grafana ID
+  string login = 2; // string ID? 
+}
+
+// Raw saved object metadata
+message RawObject {
+  // Unique ID
+  string UID = 1;
+
+  // Identify the object kind.  This kind will be used to apply a schema to the body and
+  // will trigger additional indexing behavior.
+  string kind = 2;
+
+  // Time in epoch milliseconds that the object was modified
+  int64 modified = 3;
+
+  // Who modified the object
+  UserInfo modified_by = 4;
+
+  // Content Length
+  int64 size = 5;
+
+  // MD5 digest of the Body
+  string ETag = 6;
+
+  // Raw bytes of the storage object.  The kind+schema will ensure
+  bytes body = 7;
+
+  // A small set of properties managed by raw storage layer.  The kind+schema will define val
+  map<string,string> properties = 8;
```

Comment:
I might be getting confused - how do the _properties_ relate to the _summary_?

Reply:
In the RawObject -- these are the properties managed by the storage layer.  IE the things that are real information not part of body.  

Looking at it a few days later... I think we can get rid of it (for now?) since we have the explicit properties for "comment" and "sync_time"

Reply:
removed it :)